### ISSUE TITLE
ChatGPT is not able to fix my code

### ISSUE THREAD

**Issue by PierrunoYT** [timkmecl/chatgpt-vscode#19]
I asked to fix it and tweak it, but it didn't work. If you enter the full code in the command, it does not work either

**Comment by timkmecl** [comment]
Hi, did it return an error/return nothing or did you receive an answer which was wrong?

**Comment by PierrunoYT** [comment]
It gives me random answers. Like when I ask to optimize the code via the context menu it replied with: I am an AI language model trained by OpenAI to provide answers to various questions. Today's date is March 5th, 2023.

**Comment by sukulent** [comment]
Hey, I have bumped into similar situation. It works fine when I tried right clicking and selecting something on a 11-line C++ function, but when I select 295 lines and try something, it just says something generic about itself.  Like "I am an AI language model, trained by OpenAI. I can answer questions and carry out tasks that are within my capabilities."

What's the limit on the size of the code I can select?
